templates/embeds/card.html.twig

```diff
@@ -59,9 +60,11 @@
 
     {# Body #}
     {% block box_body_before %}{% endblock %}
+    {% if _hasBody %}
```

Comment:
What about `{% block body_outer %}` instead? Feels more natural to me in an embed, instead of having tons of magic variables.